### ISSUE TITLE
Consider making labor requirements require leveled-up skill 

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#15785]
The butchery skillbook requires 'hunting labor' at level 0, this means I only have to have it discovered, I dont need to specialize in it. I think it would be better to require most labor to have the specialty, not just having read the skill scroll.

**Comment by WeaselDog** [comment]
Good idea. I see a few places where we should change the required skill but this will be a nice bonus for people who choose early gathering specialties and beyond.

**Comment by SlayksWood** [comment]
It's a bad idea. I want the first specialty to take a butchery, but I need skill scroll which make only hunter. The hunter will tell me: I won’t give you this scroll, in 2 days I’ll take a butcher myself, why do I need a competitor? So i need to take Hunting myself. 
It's also a bad idea for single player. You can play without hunting, but if you can't take Butchery you need to spend your star to Hunting.

**Comment by mirasrael** [comment]
I agree. It isn't simple change and it at least requires lot of attention to make such change and it may affect balance a lot. So can we delay this decision for post-9.0? We already have tons of issues to work in 9.0 and it doesn't looks like a simple change (taking in account potential consequences).

**Comment by WeaselDog** [comment]
We can look at research more but certainly we should make it so items that require skilled labor need specialization, i.e. concentrating ore.

**Comment by johnkslg** [comment]
Doesn't it just require changing a value from 0 to 1 in the tech tree? Part of balancing?

**Comment by mirasrael** [comment]
Because of potential problems Kirill mentioned. I.e. if we increase hewn log cost in 10 times then it will be simple number change, but will break everything

**Comment by elfl0rd** [comment]
As i think it can be ok if based on collaboration settings.
e.g. no collab - you can labor without learning
med collab - you need read scroll
high collab - you need spend star...

**Comment by WeaselDog** [comment]
We have made almost all crafting products and research require specialization. Though the skill books themselves do not require specialization, the ingredients to make them already do. 

We should look at expanding this feature as much as possible. Research and charred food require extra attention than other items, but the major of recipes should change.

**Comment by WeaselDog** [comment]
So we have spent a lot of time discussing this one. Since then we expanded on specialized labor significantly. The only part that is not changed are the research books from the original post. The logic is to allow anyone to contribute labor to research to prevent roadblocks, though people who are specialized receive additional benefits.

We have discussed some ideas to change how research works in general, but I would consider the issue closed for 9.0 and we want to have a larger discussion about how research works in general.
